src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -13,40 +15,53 @@
-                new Address("Blk 30 Lorong 3 Serangoon Gardens, #07-18"), new Country("SG"),
-                new Timezone("GMT+8"), getTagSet("colleagues", "friends")),
-            new Client(new Name("Charlotte Oliveiro"), new Phone("93210283"), new Email("charlotte@example.com"),
-                new Address("Blk 11 Ang Mo Kio Street 74, #11-04"), new Country("SG"), new Timezone("GMT+8"),
-                getTagSet("neighbours")),
-            new Client(new Name("David Li"), new Phone("91031282"), new Email("lidavid@example.com"),
-                new Address("Blk 436 Serangoon Gardens Street 26, #16-43"), new Country("SG"),
-                new Timezone("GMT+8"), getTagSet("family")),
-            new Client(new Name("Irfan Ibrahim"), new Phone("92492021"), new Email("irfan@example.com"),
-                new Address("Blk 47 Tampines Street 20, #17-35"), new Country("SG"), new Timezone("GMT+8"),
-                getTagSet("classmates")),
-            new Client(new Name("Roy Balakrishnan"), new Phone("92624417"), new Email("royb@example.com"),
-                new Address("Blk 45 Aljunied Street 85, #11-31"), new Country("SG"), new Timezone("GMT+8"),
-                getTagSet("colleagues"))
+                new Client(new Name("Alex Yeoh"), new Phone("87438807"), new Email("alexyeoh@example.com"),
+                        new Address("Blk 30 Geylang Street 29, #06-40"), new Country("SG"), new Timezone("GMT+8"),
+                        getTagSet("friends")),
+                new Client(new Name("Bernice Yu"), new Phone("99272758"), new Email("berniceyu@example.com"),
+                        new Address("Blk 30 Lorong 3 Serangoon Gardens, #07-18"), new Country("SG"),
+                        new Timezone("GMT+8"), getTagSet("colleagues", "friends")),
+                new Client(new Name("Charlotte Oliveiro"), new Phone("93210283"), new Email("charlotte@example.com"),
+                        new Address("Blk 11 Ang Mo Kio Street 74, #11-04"), new Country("SG"), new Timezone("GMT+8"),
+                        getTagSet("neighbours")),
+                new Client(new Name("David Li"), new Phone("91031282"), new Email("lidavid@example.com"),
+                        new Address("Blk 436 Serangoon Gardens Street 26, #16-43"), new Country("SG"),
+                        new Timezone("GMT+8"), getTagSet("family")),
+                new Client(new Name("Irfan Ibrahim"), new Phone("92492021"), new Email("irfan@example.com"),
+                        new Address("Blk 47 Tampines Street 20, #17-35"), new Country("SG"), new Timezone("GMT+8"),
+                        getTagSet("classmates")),
+                new Client(new Name("Roy Balakrishnan"), new Phone("92624417"), new Email("royb@example.com"),
+                        new Address("Blk 45 Aljunied Street 85, #11-31"), new Country("SG"), new Timezone("GMT+8"),
+                        getTagSet("colleagues"))
```

Comment:
this is causing the checkstyle to complain. 

Rayson and I talked pmed about this. Two dudes actually had a convo about how many spaces to have 😆 

```
____first line with 4 space indent
________thisis 8 space indent because 4+prev indent. 
```